dpgen2/exploration/selector/conf_filter.py

```diff
@@ -67,7 +67,7 @@ def check(
                     ff.check(
                         conf["coords"][ii],
                         conf["cells"][ii],
-                        conf["atom_types"],
+                        np.array([conf["atom_names"][t] for t in conf["atom_types"]]),
```

Comment:
we may want to provide both the atom types and atom names. 
or simply pass a frame of the dpdata.System. 
what do you think?

Reply:
Good idea